config/wrapper/pci_input.conf

```diff
@@ -23,6 +60,38 @@ pci_device = functions:0
 pci_devices = functions:all
 module = driver
 
+[io_iscsi_fvt]
+disk = disks:0
+htx_disks = disks:all
+disks = disks:all
+pci_device = functions:0
+pci_devices = functions:all
+module = driver
+
+[io_lsi3008_fvt]
+disk = disks:0
+htx_disks = disks:all
```

Comment:
current CR takes different disk for htx too, may be 1 or 2 devices dedicated to htx

Reply:
Sure will provide separate disk for HTX